docs/pages/deploy-a-cluster/high-availability.mdx

```diff
@@ -126,24 +126,28 @@ events:
 - Google Cloud Firestore
 - etcd (cluster state only)
 
-For Amazon DynamoDB and Google Cloud Firestore, your Teleport configuration (which
-we will describe in more detail in the [Configuration](#configuration) section)
-names a table or collection where Teleport stores cluster state and audit
-events. 
-
-The Teleport Auth Service manages the creation of any required DynamoDB tables
-or Firestore collections itself, and does not require them to exist in advance.
+For Amazon DynamoDB and Google Cloud Firestore, your Teleport configuration
+(which we will describe in more detail in the [Configuration](#configuration)
+section) names the two tables or collections where Teleport stores cluster state
+and audit events. 
 
 The Auth Service can also store cluster state in self-hosted
 [etcd](https://etcd.io/) deployments. In this case, Teleport uses namespaces
 within item keys to identify cluster state data.
 
+When you configure the Teleport Auth Service to access a table or collection,
+the Auth Service will check that the table or collection exists on startup. If
+it does not, the Auth Service attempts to create it.
+
 <Admonition title="Required permissions">
 
 In your cloud provider's RBAC solution (e.g., AWS or Google Cloud IAM), your
 Auth Service instances need permissions to read from and write to your chosen
-key/value store, as well as to create tables and collections (if your key/value
-store supports them).
+key/value store. 
+
+Unless you have a pre-existing table or collection for the Auth Service to use,
+it must also have permissions to create tables and collections (if your
```

Comment:
Line 149 renders as being within the same paragraph as line 148, so I think "it" works here. Otherwise, the sentence says, "Unless you have a pre-existing table or collection for the Auth Service to use, the Auth Service must also have permissions to create tables and collections"

Reply:
Ahh, I thought "it" might refer to another noun from the previous paragraph.

Reply:
Looking at this again now, I can definitely see how "it" could be clearer. I've suggested another wording.